### DISCUSSION TITLE
Echange lundi 30 aout

### DISCUSSION THREAD

**Discussion by gerrillari** [fabriquebeweb/scop#10] [Echange cliente]
Questions :

**Comment by sidneycarlos** [comment]
Questions/demandes pour la cliente lors du rdv du lundi 30 août:

- [x] Logo Fondespierre
- [x] Comment se passe un ordre du jour quant au déroulé lors de l'AG? Certains sujets peuvent-ils modifiés, supprimés?
- [x] Les questions seront-elles saisies une fois que l'ordre du jour sera définitif? Faut-il prévoir des possibilités d'interversion des questions?
- [x] Un sujet contient combien de questions?
- [x] Des questions peuvent-elles être rajoutées en cours d'AG?
- [x] Que se passe-t-il si le quorum en début de session n'est pas atteint?
- [x] Que se passe-t-il pour le quorum si une personne part en cours d'assemblée sans donner de pouvoir ?
- [x] Que doit-on faire des informations collectées post AG (présents, votants, quorum, quorum par question, documents...)? Pendant combien de temps les conserver?
- [x] Concernant les résultats comment doivent-il être présenter (pourcentage, valeur absolue)?
- [x] Que devrait contenir le PV?
- [x] Qui gérera la plateforme de façon centralisée ?
- [x] Comment se passerait l'intégration d'une nouvelle entreprise sur la plateforme (formulaire, automatisation)
- [x] Dans un souci d'expérience utilisateur fluide, nous prévoyons que les organisateurs d'AG puisse se connecter à la plateforme grâce à leur identifiant et leur mots de passe lié à leur compte professionnel (Microsoft, Gmail ou autre). L'administrateur de la plateforme devra avoir un accès sur chacune des plateformes. Quels seraient les freins?

**Comment by sidneycarlos** [comment]
@gerrillari @Nyviii70 @AtlasRW 
Bonjour Séverine,

Merci pour le temps que tu nous as consacré pour la réunion de ce jour.

Voici les points que tu as remontés :

Tu es globalement satisfaite de la maquette proposée mais tu as remonté quelques points sur lesquels tu attends des améliorations/ajouts :
-	Nouvelle SCOP :
o	Une nouvelle SCOP doit pouvoir demander un nouvel accès au portail via le portail les éléments qui permettront de contrôler et de valider la demande se fera par son numéro confédéral et son email
o	Une fois les vérifications réalisées, le "Super administrateur" (URSCOP ou CGSCOP) envoie à cet administrateur (SCOP) un mot de passe

-	Un sociétaire doit pouvoir donner sa procuration en amont de l’AG

-	Les pages de résultats doivent représenter les résultats « Pour », « Contre » et « Abstention »

Ci-dessous nos questions auxquelles tu as répondu :

- Est -il possible de nous fournir le logo Fondespierre ?
=> Logo « Les Scop » fourni via email par la cliente

- Comment se passe un ordre du jour quant au déroulé lors de l'AG ? Certains sujets peuvent-ils modifiés, supprimés ? 
=> Normalement non les sujets d’une AG ne sont pas modifiés malgré tout cela pourrait être un cas d’usage

- Faut-il prévoir des possibilités d'interversion des questions ? 
=> Si possible oui mais pas indispensable

- Un sujet contient combien de questions ? 
=>1 sujet = 1 question

- Des questions peuvent-elles être rajoutées en cours d'AG ? 
=> Normalement non mais ça peut être prévu dans les statuts

- Que se passe-t-il si le quorum en début de session n'est pas atteint ? 
=> Si le quorum n’est pas atteint l'AG n'a pas lieu et une seconde convocation est envoyée. Il faut malgré tout un PV avec nombre de personnes présentes, absentes, pouvoir

- Que se passe-t-il en cas de départ d'une personne en cours d'assemblée sans donner de pouvoir notamment si le quorum n'est plus respecté ?
=> Ça n'est jamais arrivé

- Que doit-on faire des informations collectées (présents, votants, quorum, quorum par question, documents...) une fois que l'AG est terminée ? Pendant combien de temps les conserver ?
=> Il s’agit d’un point à arbitrer. Solution envisageable : données supprimées après 1 mois

- Concernant les résultats comment doivent-il être présentés (pourcentage, valeur absolue) ?
=> Les résultats doivent être présentés en pourcentage et en valeur. Il faut faire figurer les « Contre », « Pour », « Abstention »

- Que devrait contenir le PV ?
=> Exemple de PV fourni par email

- Qui gérera la plateforme de façon centralisée ?
=> Il s’agit d’un point à déterminer : cela pourrait être fait au niveau URSCOP ou au niveau national, le CGSCOP disposant d’une DSI

- Comment se passerait l'intégration d'une nouvelle entreprise sur la plateforme (formulaire, automatisation) ?
=> Voir réponse ci-dessus dans les points que tu as soulevés

- Dans un souci d'expérience utilisateur fluide, nous prévoyons que les organisateurs d'AG puissent se connecter à la plateforme grâce à leur identifiant et leur mots de passe liés à leur compte professionnel (Microsoft, Gmail ou autre). L'administrateur de la plateforme devra avoir un accès sur chacune des plateformes. Quels seraient les freins ?
=> à déterminer

Nous sommes à ton écoute pour tout ajout ou modification.
Bien cordialement,
L'équipe Beweb

[logo scop CMJN copie_1 Scopika.pdf](https://github.com/fabriquebeweb/scop/files/7076066/logo.scop.CMJN.copie_1.Scopika.pdf)
[PV AGO JUIN 2020.pdf](https://github.com/fabriquebeweb/scop/files/7076068/PV.AGO.JUIN.2020.pdf)
